### ISSUE TITLE
What's the difference between metric "auroc" and sklearn.metrics.roc_auc_score?

### ISSUE THREAD

**Issue by hjzhang1018** [ThilinaRajapakse/simpletransformers#1239]
I used ClassificationModel for my dataset training. However when I called the `model.eval_model()` function on my test dataset, I got different scores of "auroc" and "roc_auc_score". Code is like below:
```
model = ClassificationModel()
model.eval_model(eval_df=test_df, output_dir=model_dir, 
                      roc_auc=sklearn.metrics.roc_auc_score, 
                      )
```
I got results like this:

>  ({'auprc': 0.5154601291728336,
>   'auroc': 0.8368392799265343,
>   'eval_loss': 4.512793393797144,
>   'fn': 20,
>   'fp': 3379,
>   'roc_auc': 0.6193144987035437,
>   'tn': 1249,
>   'tp': 620})

So I wonder what’s the difference between `"auroc"` and `"roc_auc"`. The former seems like to be a default metric but I didn’t find any introductions of it. Besides, I'm not very clear about `"auprc"` as well. Does it equal to `"sklearn.metrics.precision_recall_curve"`? Can anyone please explain this to me? I appreciate your help!
